src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractPackage.cs

```diff
@@ -1,23 +1,24 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
 using System.Threading;
-using System.Threading.Tasks;
 using System.Windows.Threading;
 using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
 using Microsoft.CodeAnalysis.Utilities;
 using Microsoft.VisualStudio.Shell;
+using Task = System.Threading.Tasks.Task;
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.LanguageService
 {
-    internal abstract class AbstractPackage : Package
+    internal abstract class AbstractPackage : AsyncPackage
```

Comment:
I would expect that F# or TypeScript are inheriting from this type and thus this is a breaking change for them. Not that this isn't a virtuous thing, but we'll have to coordinate/test accordingly.

Reply:
This is still a concern. I do see legitimate breakage here: 

https://github.com/Microsoft/visualfsharp/blob/eaff32008b8714f27d1a3db49c89ebde6a225da8/vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs#L376-L377